packages/core/src/constants.ts

```diff
@@ -27,6 +28,7 @@ export const CHAIN_SPECIFIERS = {
     [CHAIN_IDS.ArbitrumOne]: "arbitrum-one",
     [CHAIN_IDS.ArbitrumSepolia]: "arbitrum-sepolia",
     [CHAIN_IDS.BaseSepolia]: "base-sepolia",
+    [CHAIN_IDS.BaseMainnet]: "base-mainnet",
```

Comment:
super small nit: can we order so it's mainnet first i.e. Arb One, Arb Sep, Base Mainnet, Base Sep.